inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -459,11 +518,40 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 			}
 		},
 		async () => {
-			const repoMeta = await repo()?.getMeta()
+			const repoMeta = await repo()!.getMeta()
```

Comment:
we should not do ! here. repo should be an argument from the previous function insteafd of unwrapping repo() signal again in the effect, i think

Reply:
@NiklasBuchfink I changed this to a bang (!) because it should be available at this state and i was fine with this for now - we discussed this on the call. but @janfjohannes also noticed that :) please decide if we should change that in this pr

Reply:
I think @janfjohannes is right, we can pass it as an argument as it is done in the previous function. This should eliminate the need for the bang.

Reply:
Last push should have tackled this one